torch/csrc/jit/script/schema_matching.cpp

```diff
@@ -239,14 +239,14 @@ static bool varargsCanBeUsedAsList(
   bool is_last_argument = arg_index + 1 == schema.arguments().size() ||
       schema.arguments()[arg_index + 1].kwarg_only();
 
-  // The formal must be a list
-  bool argument_is_list = arg.type()->kind() == TypeKind::ListType;
```

Comment:
https://github.com/pytorch/pytorch/blob/master/torch/csrc/utils/python_arg_parser.cpp#L490

Specifies that this functionality is only supposed to work for intlists, but looking at test failures it looks like this is also being used for `broadcast_tensors`